ansible/roles/slurm_exporter/tasks/install.yml

```diff
@@ -0,0 +1,72 @@
+- name: Create slurm exporter group
+  ansible.builtin.group:
+    name: "{{ slurm_exporter_group }}"
+    state: present
+    system: true
+
+- name: Create user
+  ansible.builtin.user:
+    name: "{{ slurm_exporter_user }}"
+    groups: "{{ slurm_exporter_group }}"
+    append: true
+    shell: /usr/sbin/nologin
+    system: true
+    create_home: false
+    home: /
+
+- name: Install GCC
+  ansible.builtin.dnf:
+    name: gcc
+    state: latest
+
+- name: Install go
+  ansible.builtin.unarchive:
+    remote_src: yes
+    src: "https://go.dev/dl/go{{ slurm_exporter_go_version }}.linux-{{ slurm_exporter_go_arch }}.tar.gz"
+    dest: /usr/local
```

Comment:
Could this go to `/tmp/{{ slurm_exporter_godir }}`, then be removed after being added to the PATH for the `make` stage?
Seems like quite a big thing to leave hanging around in an image just to build the exporter!

Reply:
Hopefully you should be able to use https://github.com/stackhpc/prometheus-slurm-exporter/releases/download/0.20/prometheus-slurm-exporter-0.20-2.el8.x86_64.rpm to install the built exporter now, along with a user and listening on 9341.